src/java.base/share/classes/java/lang/reflect/ClassFileFormatVersion.java

```diff
@@ -0,0 +1,350 @@
+     * JEP 181: Nest-Based Access Control</a>
+     */
+    RELEASE_11(55),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 12.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se12/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 12 Edition</cite></a>
+     */
+    RELEASE_12(56),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 13.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se13/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 13 Edition</cite></a>
+     */
+    RELEASE_13(57),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 14.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se14/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 14 Edition</cite></a>
+     */
+    RELEASE_14(58),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 15.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se15/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 15 Edition</cite></a>
+     * @see <a href="https://openjdk.java.net/jeps/371">
+     * JEP 371: Hidden Classes</a>
+     */
+    RELEASE_15(59),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 16.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se16/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 16 Edition</cite></a>
+     */
+    RELEASE_16(60),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 17.
+     *
+     * Additions in this release include sealed classes and
+     * restoration of always-strict floating-point semantics.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se17/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 17 Edition</cite></a>
+     * @see <a href="https://openjdk.java.net/jeps/306">
+     * JEP 306: Restore Always-Strict Floating-Point Semantics</a>
+     * @see <a href="https://openjdk.java.net/jeps/409">
+     * JEP 409: Sealed Classes</a>
+     */
+    RELEASE_17(61),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 18.
+     *
+     * @see <a
+     * href="https://docs.oracle.com/javase/specs/jvms/se18/html/index.html">
+     * <cite>The Java Virtual Machine Specification, Java SE 18 Edition</cite></a>
+     */
+    RELEASE_18(62),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 19.
+     */
+    RELEASE_19(63),
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 20.
+     */
+    RELEASE_20(64);
+
+    private int major;
```

Comment:
Could be final?

Reply:
Sure; that is a better expression of the intent of the field. Will change before pushing; thanks.